### DISCUSSION TITLE
Problem with trigger SMCB-Authentication

### DISCUSSION THREAD

**Discussion by mge-medisoftware** [Digitaler-Impfnachweis/certification-apis#32] [Q&A] [answered]
Starting Trigger SMCB-Authentication returns "500 Server Error" using "client_id=cert-auth"
With new "client_id=user-access-ti" result of request is 400 Bad Request" with kc-error-message "Client not Found" 

OIDC Authorization Endpoint https://keycloak-cert-auth.dev.ibmega.net/auth/realms/master/protocol/openid-connect/auth

Complete authentication worked last week with "client_id=cert-auth", what has been changed?

**Comment by thinkberg** [comment]
@oliver-steinbrecher I think this is your terrain.

**Comment by fue-medatixx** [comment, marked as answer]
The Status 500 Internal Server Error is resolved. 
It works for us with "client_id=cert-auth" again.

**Reply by oliver-steinbrecher** [reply]
Issue should be resolved.

**Reply by oliver-steinbrecher** [reply]
In both environments 
1. https://keycloak-cert-auth.dev.ibmega.net
2.https://id.ru.impfnachweis.info
